models/issue_dependency.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
+
+import (
+	"time"
+)
+
+// IssueDependency is connection request for receiving issue notification.
+type IssueDependency struct {
+	ID           int64     `xorm:"pk autoincr"`
+	UserID       int64     `xorm:"UNIQUE(watch) NOT NULL"`
```

Comment:
This unnecessary. issue notifications are tracked elsewhere, use a JOIN if necessary

Reply:
Yup, you're right, it probably is there because I copied it in the beginning 😀 , my fault...

Reply:
Fixed it.